Web/Edubase.Web.UI/Areas/Establishments/Controllers/AcademyOpeningsController.cs

```diff
@@ -38,9 +36,23 @@ public class AcademyOpeningsController : Controller
             _establishmentWriteService = establishmentWriteService;
         }
 
+        [HttpGet, Route("16-19-secure-academy-openings", Name = "ManageSecureAcademy16To19Openings")]
+        public Task<ActionResult> ManageSecureAcademy16To19Openings(int skip = 0, string sortBy = "OpenDate-desc")
+        {
+            //secure 16-19 academy establishment type Id is 46
+            return Task.FromResult<ActionResult>(RedirectToAction(nameof(ManageAcademyOpenings),
+                new
+                {
+                    skip,
+                    sortBy,
+                    isSecure16To19User = SecureAcademyUtility.EncryptValue(true.ToString()),
+                    establishmentTypeId = SecureAcademyUtility.EncryptValue("46")
+                }));
+        }
+
         [HttpGet, Route("academy-openings", Name = "ManageAcademyOpenings")]
-        // public ActionResult Index() => View();
-        public async Task<ActionResult> ManageAcademyOpenings(int skip = 0, string sortBy = "OpenDate-desc")
+        public async Task<ActionResult> ManageAcademyOpenings(int skip = 0, string sortBy = "OpenDate-desc",
+            string isSecure16To19User = null, string establishmentTypeId = null)
```

Comment:
Do isSecure16To19User and establishmentTypeId need to be type string ?

Reply:
No, not necessarily. I found it easier to work it in this case. I feel they somewhat represent some of the 
states a bit better from the vue side